app/data/test_interstitial_page.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
I think we should aim to create the simplest possible schemas to demonstrate features. Using a simple text input per page rather than a radio options would be better here.